src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx

```diff
@@ -0,0 +1,157 @@
+import FormFieldThemeTopicSelect from '@admin/components/form/FormFieldThemeTopicSelect';
+import { Theme } from '@admin/services/themeService';
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import ButtonText from '@common/components/ButtonText';
+import { FormFieldset, FormFieldSelect } from '@common/components/form';
+import Form from '@common/components/form/Form';
+import FormFieldTextInput from '@common/components/form/FormFieldTextInput';
+import ModalConfirm from '@common/components/ModalConfirm';
+import useFormSubmit from '@common/hooks/useFormSubmit';
+import useToggle from '@common/hooks/useToggle';
+import { PublicationSummary } from '@common/services/publicationService';
+import { mapFieldErrors } from '@common/validation/serverValidations';
+import Yup from '@common/validation/yup';
+import { Formik } from 'formik';
+import React from 'react';
+
+export interface FormValues {
+  supersededById?: string;
+  title: string;
+  topicId: string;
+}
+
+interface Props {
+  canUpdatePublicationSupersededBy?: boolean;
+  canUpdatePublicationTitle?: boolean;
+  initialValues: FormValues;
+  publications?: PublicationSummary[];
+  themes?: Theme[];
+  onCancel: () => void;
+  onSubmit: (values: FormValues) => void;
+}
+
+const PublicationDetailsForm = ({
+  canUpdatePublicationSupersededBy = false,
+  canUpdatePublicationTitle = false,
+  initialValues,
+  publications,
+  themes,
+  onCancel,
+  onSubmit,
+}: Props) => {
+  const [showConfirmModal, toggleConfirmModal] = useToggle(false);
+
+  const errorMappings = [
+    mapFieldErrors<FormValues>({
+      target: 'title',
+      messages: {
+        SlugNotUnique: 'Choose a unique title',
+      },
+    }),
+  ];
+
+  const validationSchema = Yup.object<FormValues>({
+    title: Yup.string().required('Enter a title'),
+    topicId: Yup.string().required('Choose a topic'),
+  });
+
+  const handleSubmit = useFormSubmit((values: FormValues) => {
+    onSubmit(values);
+  }, errorMappings);
+
+  const id = 'publicationDetailsForm';
+
+  return (
+    <Formik<FormValues>
+      enableReinitialize
+      initialValues={{
+        ...(initialValues ?? {
+          theme: '',
+          title: '',
+          topicId: '',
+        }),
+      }}
+      validationSchema={validationSchema}
+      onSubmit={handleSubmit}
+    >
+      {form => (
+        <>
+          <Form id={id}>
+            <FormFieldset id="details" legend="Publication details">
+              {canUpdatePublicationTitle && (
```

Comment:
It doesn't actually make a lot of sense why we'd be able to update the publication title but still be able to change the theme and topic as well.

Might want to check with Cam on this, but would suggest that if you can't change the title, you also shouldn't be able to change the theme or topic. 

I'd probably want to go further and change the backend permission naming to something more appropriate like `canUpdatePublicationDetails`. Having permissions for individual fields like the title is totally overkill. We don't have to do this as part of this work, but would suggest we ticket it up as an improvement / tech debt.

Reply:
As discussed in slack, have wrapped `canUpdatePublicationTitle` around the theme and topic, as well as title, and will raise a ticket to tidy the permissions up.